e2e/benchmarks/model_config.js

```diff
@@ -198,7 +198,10 @@ const benchmarks = {
     type: 'GraphModel',
     inputSizes: [128, 257, 512, 1024],
     architectures: ['MobileNetV1', 'ResNet50'],
-    load: async (inputResolution = 128, modelArchitecture = 'MobileNetV1') => {
+    inputTypes: ['image', 'tensor'],
```

Comment:
Can you also add imageBitmap as another option?

Reply:
For below  models:
```
MobileNet, Bodypix, BlazeFace, PoseNet, coco-ssd, deeplab
```
They only support below input type, ImageBitmap is not includes:
```
tf.Tensor|ImageData|HTMLImageElement|HTMLCanvasElement|HTMLVideoElement
```
So before we add ImageBitmap support, we need to change the models input(from tfjs-models) first.

Reply:
Make sense for me. Thanks.